### PR TITLE
Move whitespace

### PR DESCRIPTION
**Pull request by graichen**

### CODE REVIEW

File: Day1/PythonEditing.md [hunk 1]
```diff
@@ -1,28 +1,32 @@
 |[< Previous (HelloWorld)](HelloWorld.md) | [Day1](../README.md)| [Next (Variables) > ](Variables.md) |
 |----|----|----|
 # Python specifics
```
Comment (sarcar) [inline Day1/PythonEditing.md]: Suggest title be Python Editing for consistency

**Review by sarcar — COMMENTED** [review]
Suggest title be Python Editing for PythonEditing.md

File: Day1/PythonEditing.md (hunk 1 above)
Comment (graichen) [reply Day1/PythonEditing.md]: changed...

**Comment by graichen** [comment]
ok really made the change now!

**Comment by sarcar** [comment]
Thanks Cathy !

**Pull request merged**